api-scanner/src/main/java/net/corda/plugins/ScanApi.java

```diff
@@ -332,18 +341,25 @@ private void writeMethods(PrintWriter writer, List<MethodInfo> methods) {
                         && isValid(method.getAccessFlags(), METHOD_MASK) // Excludes bridge and synthetic methods
                         && !hasInternalAnnotation(method.getAnnotationNames()) // Excludes methods annotated as @CordaInternal
                         && !isKotlinInternalScope(method)) {
-                    writer.append("  ").println(filterAnnotationsFor(method));
+                    for(String annotation: getAnnotationsFor(method)) {
```

Comment:
I suppose I'm thinking of something like this:
```
println(writer, filterAnnotationsFor(method), "  ");
```
where the third parameter is the indentation string that we need to prefix each line with. I hope this would be prettier than a `toString()`-like function that is aware of both indentation-width and line separators :nauseated_face:.

Reply:
Heh, or maybe even make a `PrintWriter` sub-class with the following extra methods:
```java
void println(Methodinfo method, String prefix);
void println(FieldInfo field, String prefix);
```
I think `ScanApi.java` is becoming rather "code-heavy", so maybe a custom `PrintWriter` class would help?